src/main/cliapp/src/containers/allelesPage/symbols/SymbolFormEditor.js

```diff
@@ -0,0 +1,110 @@
+import { Button } from "primereact/button";
+import { FormTableWrapper } from "../../../components/FormTableWrapper";
+import { useRef } from "react";
+import { SymbolFormTable } from "./SymbolFormTable";
+
+export const SymbolFormEditor = ({ labelColumnSize, state, dispatch }) => {
```

Comment:
Can we rename this (and the file) to SymbolForm for consistency with the other tables on the detail page?

Reply:
renamed

Reply:
@abecerra Looks like you didn't commit the renamed file?

Reply:
It should be there, I had to rebase the branch with alpha, I'll fix the  props  values!